src/AppBundle/Controller/ProcurationManagerController.php

```diff
@@ -70,31 +70,32 @@ public function requestsListAction(Request $request, $page): Response
      */
     public function proposalsAction(Request $request): Response
     {
-        $proxiesRepository = $this->getDoctrine()->getRepository(ProcurationProxy::class);
-        $filters = (new ProcurationParametersFilter())->handleRequest($request);
-
-        return $this->render('procuration_manager/proposals.html.twig', [
-            'proxies' => $proxiesRepository->findManagedBy($this->getUser(), 1, self::PER_PAGE, $filters),
-            'totalCount' => $proxiesRepository->countManagedBy($this->getUser(), $filters),
-            'filters' => $filters,
-            'filterqQueryString' => http_build_query($request->query->all()),
-        ]);
+        try {
+            $filters = ProcurationProxyProposalFilters::fromQueryString($request);
+
+            $manager = $this->get('app.procuration.manager');
+
+            return $this->render('procuration_manager/proposals.html.twig', [
+                'proxies' => $manager->getProcurationProxyProposals($this->getUser(), $filters),
+                'total_count' => $manager->countProcurationProxyProposals($this->getUser(), $filters),
+                'filters' => $filters,
+            ]);
+        } catch (ProcurationException $e) {
```

Comment:
I meant `ProcurationException` is only throw from the method `fromQueryString` right ?